docs/README.md

```diff
@@ -49,7 +49,6 @@ title: Metabase documentation
 - [SQL templates](./questions/native-editor/sql-parameters.md)
 - [Using results to ask new questions](./questions/native-editor/referencing-saved-questions-in-queries.md)
 - [SQL snippets](./questions/native-editor/sql-snippets.md)
-- [SQL snippet permissions](./questions/native-editor/snippet-permissions.md)
```

Comment:
Could keep the link here since the content is still SQL-specific, and just update the path?

Oh, also "SQL snippet folder permissions" for consistency!